transport-native-io_uring/src/main/c/io_uring.h

```diff
@@ -48,7 +48,10 @@ struct io_uring_sqe {
 	};
 	__u32	len;		/* buffer size or number of iovecs */
 	union {
-		__kernel_rwf_t	rw_flags;
+		// IMPORTANT:
+		// We explicit use 'int __bitwise' here and not ' __kernel_rwf_t' as this may not be present in the
+		// kernel headers that are used on the system.
+		int __bitwise	rw_flags;
```

Comment:
interesting I thought it would even be present on old kernel version like 4.x

Reply:
we compile our linux based releases on centos 6 for max compatibility. As far as I know these not ship 4.x kernels. So this makes sense I guess :)

Reply:
Also I checked and this only exists in 5.x kernels.